### PR TITLE
console: use requestAction instead of fetch

### PR DESCRIPTION
**Pull request by soorajshankar**
### Description

Removed all direct usage of `fetch` and used redux action `requestAction` or `requestActionPlain`

### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Console

### Related Issues
close https://github.com/hasura/graphql-engine-internal/issues/475
### Solution and Design

code refactoring 

### Steps to test and verify
need to test all API call for the touched modules.

#### Catalog upgrade
Does this PR change Hasura Catalog version?
- [x] No

#### Metadata
Does this PR add a new Metadata feature?
- [x] No

#### GraphQL
- [x] No new GraphQL schema is generated

#### Breaking changes

- [x] No Breaking changes

### CODE REVIEW

File: console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import React from 'react';
```
Comment (beerose) [inline console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js:1]: Looks like there are two possibly same files:
- console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js
- console/src/components/Services/ApiExplorer/Analyzer/AnalyseButton.js

And I think we should stick to American spelling (Analyze) as it's more common and already used in many places.

File: console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 import PropTypes from 'prop-types';
-import QueryAnalyzer from './QueryAnalyzer';
+import QueryAnalyser from './QueryAnalyzer';
```
Comment (beerose) [inline console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js:3]: Ditto

File: console/src/components/Services/ApiExplorer/GraphiQLWrapper/GraphiQLWrapper.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import GraphiQLErrorBoundary from './GraphiQLErrorBoundary';
 import OneGraphExplorer from '../OneGraphExplorer/OneGraphExplorer';
-import AnalyzeButton from '../Analyzer/AnalyzeButton';
+import AnalyseButton from '../Analyzer/AnalyzeButton';
```
Comment (beerose) [inline console/src/components/Services/ApiExplorer/GraphiQLWrapper/GraphiQLWrapper.js:8]: Ditto

File: console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js (hunk 1 above)
Comment (soorajshankar) [reply console/src/components/Services/ApiExplorer/Analyzer/AnalyzeButton.js:1]: Oops, i wanted to do the same, seems like `AnalyseButton` didn't get deleted 
finding now

**Pull request merged**